content/en/security/application_security/api-inventory/_index.md

```diff
@@ -42,20 +45,15 @@ The following library versions are compatible with API Security Inventory. [Remo
 
 ## How it works
 
-API Inventory leverages the Datadog tracing library with ASM enabled to gather security metadata about API traffic, including the API schema, types of sensitive data processed, and the authentication scheme.
+API Inventory leverages the Datadog tracing library with ASM enabled to gather security metadata about API traffic, including the API schema, types of sensitive data processed, and the authentication scheme. API information is evaluated per endpoint, every 30 seconds, which should ensure minimal performance impact.
 
 API Inventory Security uses [Remote Configuration][4] to manage and configure scanning rules that detect sensitive data and authentication.
 
 The following risks are calculated for each endpoint:
 
-### Service in production
-
-The `env` tag is checked for patterns that frequently represent non-production environments. For example, if it detects values of `dev`, `alpha`, `beta`, `sandbox`, or similar, it marks the environment as non-production. All other environments are marked as production.
-
-
 ### Service under attack
 
-This risk is detected for API endpoints that have experienced [attacks][2] within the last week.
+This risk is detected for API endpoints that have experienced [attacks][2] within the last day. You can find this week's security of the endpoint in the endpoint's side panel.
```

Comment:
Not sure I understand the second part of the sentence.
For the first one. This risk is actually detection for API endpoints that have received traffic within the last day. It's not linked to attacks anymore

Reply:
ah, you mean service under attack is not linked to attack attempts any more?

Reply:
I removed the second line and changed the first